tests/integration/targets/ipam_ip_space/tasks/main.yml

```
@@ -620,6 +620,30 @@
             _federated_realm5.id
             ] | sort)
 
+    - name: "Create an IP space with dhcp options"
```

```suggestion
    - name: "Create an IP space with DHCP Options"
```
Please make this change for all the tasks

---

done

---

Please make this change for other tasks also

---

done